Cargo.toml

```diff
@@ -11,6 +11,22 @@ members = [
     "boa_examples",
 ]
 
+[workspace.package]
+edition = "2021"
+version = "0.15.0"
+rust-version = "1.64"
+authors = ["boa-dev"]
+repository = "https://github.com/boa-dev/boa"
+license = "Unlicense/MIT"
+description = "Boa is a Javascript lexer, parser and Just-in-Time compiler written in Rust. Currently, it has support for some of the language."
+
+[workspace.dependencies]
+boa_engine = { version = "0.15.0", path = "boa_engine" }
+boa_interner = { version = "0.15.0", path = "boa_interner" }
```

Comment:
What happens if version is not specified?

Reply:
Crates.io rejects publishing the crate